### PR TITLE
CASM-4555

### PR DESCRIPTION
**Pull request by Srinivas-Anand-HPE**
# Description

CSM Upgrade through IUF: Documentation

Note: Diagram has to be updated for 1.6 in seperate Jira

Remove reference to CSM WLM backup from upgrade documentation 

Resolves  :

[CASM-4555](https://jira-pro.it.hpe.com:8443/browse/CASM-4555) CSM Upgrade through IUF: Documentation

[CASM-4822](https://jira-pro.it.hpe.com:8443/browse/CASM-4822) Remove reference to CSM WLM backup from upgrade documentation 
 
# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

### CODE REVIEW

File: operations/iuf/workflows/management_rollout.md [hunk 1]
```diff
@@ -47,6 +51,8 @@ procedures based on whether or not CSM is being upgraded:
 
 ### 3.1 `management-nodes-rollout` with CSM upgrade
 
+**`NOTE`** This subsection is optional and can be skipped if Install or upgrade additional products with IUF.
```
Comment (don-bahls-hpe) [inline operations/iuf/workflows/management_rollout.md]: This statement is not true.   If USS or SHS change you need to do `management-nodes-rollout`.

File: operations/iuf/workflows/management_rollout.md [hunk 2]
```diff
@@ -426,7 +473,7 @@ The worker canary node can be any worker node and does not have to be `ncn-w001`
 1. (`ncn-m001#`) Use `kubectl` to remove the `iuf-prevent-rollout=true` label from the canary node.
 
     ```bash
-    kubectl label nodes "${WORKER_CANARY}" --overwrite iuf-prevent-rollout-
+    kubectl label nodes "${WORKER_CANARY}" --overwrite iuf-prevent-rollout
```
Comment (don-bahls-hpe) [inline operations/iuf/workflows/management_rollout.md]: The `-` at the end of the line is required to remove the label.   This command should be what it was before:  

```
kubectl label nodes "${WORKER_CANARY}" --overwrite iuf-prevent-rollout-
```

File: operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md [hunk 3]
```diff
@@ -13,81 +13,13 @@ on the new images provided by the CSM product and customized by the additional H
...
-
-   Follow these IUF instructions in order:
-
-   1. [Deploy product](deploy_product.md)
-   1. [Validate deployment](validate_deployment.md)
-
-1. Managed rollout
-
-   Follow the IUF [Managed rollout](managed_rollout.md) instructions.
+1. This option alternates between CSM upgrade instructions that do not utilize the IUF
```
Comment (don-bahls-hpe) [inline operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md]: I would drop the "the" before "IUF"
```suggestion
1. This option alternates between CSM upgrade instructions that do not utilize IUF
```

File: operations/iuf/workflows/upgrade_csm_iuf_additional_products_with_iuf.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
+# Upgrade CSM and additional products with IUF
+
+**Note: From CSM 1.6 , CSM supports upgrade through IUF. So, all the CSM specific steps mentioned in [Upgrade CSM manually and additional products with IUF](upgrade_csm_manual_and_additional_products_with_iuf.md) are now part of CSM Upgrade with IUF.**
```
Comment (don-bahls-hpe) [inline operations/iuf/workflows/upgrade_csm_iuf_additional_products_with_iuf.md]:
```suggestion
**Note: From CSM 1.6 , CSM supports upgrade through IUF.   All CSM specific steps mentioned in [Upgrade CSM manually and additional products with IUF](upgrade_csm_manual_and_additional_products_with_iuf.md) are now part of CSM Upgrade with IUF.**
```

File: operations/iuf/workflows/management_rollout.md (hunk 2 above)
Comment (Srinivas-Anand-HPE) [reply operations/iuf/workflows/management_rollout.md]: Modified required changes

File: operations/iuf/workflows/management_rollout.md (hunk 1 above)
Comment (Srinivas-Anand-HPE) [reply operations/iuf/workflows/management_rollout.md]: Thanks for your comment . Note may not be added to 3.1 `management-nodes-rollout` with CSM upgrade
and 3.2 `management-nodes-rollout` without CSM upgrade

**Review by rustydb — APPROVED** [review]

**Review by nrockershousen — APPROVED** [review]

**Review by don-bahls-hpe — APPROVED** [review]

**Pull request merged**